pyomo/contrib/incidence_analysis/interface.py

```diff
@@ -866,35 +884,48 @@ def remove_nodes(self, nodes, constraints=None):
 
         Parameters
         ----------
-        nodes: list
-            VarData or ConData objects whose columns or rows will be
-            removed from the incidence matrix.
+        variables: list
+            VarData objects whose nodes will be removed from the incidence graph
         constraints: list
-            VarData or ConData objects whose columns or rows will be
-            removed from the incidence matrix.
+            ConData objects whose nodes will be removed from the incidence graph
+
+        .. note::
+
+           **Breaking change in Pyomo vTBD**
```

Comment:
Do you want to add in a deprecation warning? It would be "louder" to the users.

Reply:
I've updated `remove_nodes` to allow the old API by manually sorting variables/constraints and raising a deprecation warning if either is provided in the wrong list.